internal/resources/types.go

```diff
@@ -75,7 +75,8 @@ func FromDBType(dbResource *db.Resource) *Resource {
 		resource.Website = &dbResource.Website.String
 	}
 	if dbResource.CertifiedAt.Valid {
-		resource.CertifiedAt = &dbResource.CertifiedAt.String
+		certifiedAt := dbResource.CertifiedAt.Time.Format("2006-01-02T15:04:05.999Z07:00")
```

Comment:
The formatting we have in rails is not the RFC 3339 standard so it's custom here, I'll make this a constant later.

Reply:
Just curious, what is the discrepancy between Rails' format and RFC 3339? Does Rails emit ISO 8601 instead of RFC 3339? The two standards are similar, and while there are date strings that are legal in both, there are some strings that are legal in one but not the other.